src/lib/components/Codelab/index.js

```diff
@@ -29,7 +32,21 @@ class Codelab extends BaseElement {
 
     this.glitch = "";
     this.path = "index.html";
-    this.iframeEnabled = false;
+    this._isDesktop = false;
```

Comment:
Shouldn't this be initialized based on `this._mql.matches` instead of calling `this._toggleDesktop();` in `connectedCallback`?

Reply:
`render()` is only called between `connectedCallback` and `disconnectedCallback`, so the value of this variable doesn't matter except during that time. I'll add a comment.

Reply:
Good!